src/grizzly/services/webtransport/core.py

```diff
@@ -0,0 +1,126 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import asyncio
+import os
+from asyncio import AbstractEventLoop
+from logging import getLogger
+from pathlib import Path
+from platform import system
+from threading import Thread
+from typing import Optional
+
+from aioquic.asyncio import serve  # type: ignore[attr-defined]
+from aioquic.h3.connection import H3_ALPN
+from aioquic.quic.configuration import QuicConfiguration
+
+from ..base import BaseService
+from .wpt_h3_server import webtransport_h3_server
+from .wpt_h3_server.webtransport_h3_server import (
+    SessionTicketStore,
+    WebTransportH3Protocol,
+    _connect_to_server,
+)
+
+# Override global _doc_root
+webtransport_h3_server._doc_root = os.path.join(  # pylint: disable=protected-access
```

Comment:
You can use `from os.path import...` to cleanup this line a bit.

Reply:
or `str(Path(__file__).parent / "wpt_h3_server", "handlers")`

Reply:
Fixed in https://github.com/MozillaSecurity/grizzly/pull/484/commits/3bdb373a47e8f76f80eb85a153646745f9d3e1b9